staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -4555,7 +4562,7 @@ func (c *Cloud) updateInstanceSecurityGroupsForLoadBalancer(lb *elb.LoadBalancer
 			if !changed {
 				klog.Warning("Allowing ingress was not needed; concurrent change? groupId=", instanceSecurityGroupID)
 			}
-		} else {
```

Comment:
Before if it wasn't changed you got a warning in the logs that it wasn't changed. Do we want an else clause just to indicate a change wasn't attempted because it involved an elb security group?

Reply:
That's a warning for a concurrent change. Not related to any decision-making, but based on the status of objects in the AWS API.

I believe that there is no need to change anything.